shared/message/vision/VisualMesh.proto

```diff
@@ -24,18 +24,24 @@ package message.vision;
 import "google/protobuf/timestamp.proto";
 import "Matrix.proto";
 
+/**
+ * Contains the details of a Visual Mesh™ measurement
+ */
 message VisualMesh {
     message Row {
         float phi      = 1;
         int32 segments = 2;
     }
-    uint32                    camera_id       = 1;
-    fmat                      rays            = 2;
-    imat                      neighbourhood   = 3;
-    fmat                      classifications = 4;
-    fmat                      coordinates     = 5;
-    repeated int32            indices         = 6 [packed = true];
-    mat4                      Hcw             = 7;
-    repeated Row              mesh            = 8;
-    google.protobuf.Timestamp timestamp       = 9;
+    /// The camera which took the measurements
+    uint32 camera_id = 1;
+    /// Unit vectors with the origin at the camera position, which represent rays of light
+    fmat           rays            = 2;
```

Comment:
Unit vectors are in world space at camera origin. They represent points in the mesh (vector projected from optical axis through a pixel coordinate and on to the observation plane). This matrix has shape `3 x N`, where `N` is the number of rays in the Visual Mesh